Packs/JARM/Integrations/JARM/JARM.py

```diff
@@ -0,0 +1,158 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from urllib.parse import urlparse
+from ipaddress import ip_address
+from typing import Dict, Tuple, Any
+from jarm.scanner.scanner import Scanner
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+DEFAULT_PORT = 443
+
+""" CLIENT CLASS """
+
+
+class Client:
+    def jarm_fingerprint(self, host: str, port: int) -> Tuple[str, str, int]:
+        return Scanner.scan(host, port)
+
+
+""" HELPER FUNCTIONS """
+
+
+def parse_hostname(hostname: str, port: Optional[int]) -> Dict[str, Any]:
+    """
+    Parses a target hostname. Supports multiple ipv4/fqdn with and without port formats.
+    """
+    target: Dict[str, Any] = {}
+    if not hostname.startswith('https://'):
+        hostname = 'https://' + hostname
+
+    parsed_url = urlparse(hostname)
+    if port:
+        target['port'] = port
+    elif parsed_url.port:
+        target['port'] = parsed_url.port
+    else:
+        target['port'] = DEFAULT_PORT
+
+    try:
+        ip = ip_address(parsed_url.hostname)
+        target['target_host'] = str(ip)
+        target['target_type'] = 'ip'
+    except ValueError:
+        target['target_host'] = parsed_url.hostname
+        target['target_type'] = 'fqdn'
+
+    return target
+
+
+""" COMMAND FUNCTIONS """
+
+
+def test_module(client: Client) -> str:
+    return "ok"
+
```

Comment:
Please test here your `jarm_fingerprint` command

Reply:
Let's discuss this during the review.

Reply:
resolved